README.md

```diff
@@ -3,36 +3,28 @@
 A collection of reusable [React](https://reactjs.org/) components, patterns, and design guidelines we use at
 [Ethos](https://ethoslife.com) to build our digital products.
 
-## Prequisites
+## Setup
 
-See the `package.json` for node `engines`—you need to have that version of node installed via `nvm` specifically:
+1. See `package.json` for node `engines` – you need to have that version of node installed specifically, e.g. via `nvm`:
 
 ```shell
-nvm install v10.16.3 # then do `nvm list` to ensure version is available
+nvm install v10.16.3  # then do `nvm list` to ensure version is available
+nvm use               # use the version specified in `.nvmrc`, if installed
 ```
 
-## Installing fonts for local development
+2. Download `fonts.zip` (pinned to #dev-design-system) and unzip it.
```

Comment:
I think this is now in Google Drive since Slack deletes pinned stuff after a couple of weeks.

Reply:
https://drive.google.com/drive/u/0/folders/1hvAAEUWEsz2Hq-Wmj09xOvCYeixkJ4_c

Reply:
Fixed.